File: Week 6/templates/C/C_weightofwords.py
#That but in python:

# There's some non DP approaches to this problem
# If you wanted to get started w/ a DP approach, I'd recommend something like this:
L, W = 40 + 5, 1000 + 5
visited = [[False for _ in range(W)] for _ in range(L)]
ans = ""
def solve(i, j):
    # TODO: implement this function
    # Return true if we find a word of length i with weight j
    return False

def weightofwords(l, w):
    if not solve(l, w):
        return "impossible"
    return ans

l, w = map(int, input().split())
print(weightofwords(l, w))

File: Week 6/templates/C/C_weightofwords.cpp
#include <bits/stdc++.h>
using namespace std;

// There's some non DP approaches to this problem
// If you wanted to get started w/ a DP approach, I'd recommend something like this:
const int L = 40 + 5, W = 1000 + 5;
bool visited[L][W];
string ans;
bool solve(int i, int j){
    // TODO: implement this function
    // Return true if we find a word of length i with weight j
    return false;
}
string weightofwords(int l, int w){
    if(!solve(l, w)){
        return "impossible";
    }
    return ans;
}

int main(){
    int l, w;
    cin >> l >> w;
    cout << weightofwords(l, w) << '\n';
}